pkg/controller/release/testmain_test.go

```diff
@@ -1,28 +0,0 @@
-package release
-
-import (
-	"log"
-	"os"
-	"testing"
-
-	"k8s.io/helm/pkg/repo/repotest"
-)
-
-var chartRepoURL string
-
-func TestMain(m *testing.M) {
-	srv, hh, err := repotest.NewTempServer("testdata/*.tgz")
```

Comment:
I did not even know we did this, but makes me happy to see it gone <3

Reply:
My bad: apparently, it was an easy way to plug it back when I was working on release controller, and never fixed it since then. Clearly there is no place for this kind of hacks in unit testing.